gui/doc/table.csv

```diff
@@ -34,7 +34,7 @@ filter,bool,FALSE,"Indicates, if True, that all columns can be filtered."
 filter[<i>col_name</i>],bool,FALSE,"Indicates, if True, that the indicated column can be filtered."
 nan_value,str,"""""",The replacement text for NaN (not-a-number) values.
 nan_value[<i>col_name</i>],str,"""""",The replacement text for NaN (not-a-number) values for the indicated column.
-editable,dynamic(bool),FALSE,"Indicates, if True, that all columns can be edited."
+editable,dynamic(bool),True,"Indicates, if True, that all columns can be edited."
```

Comment:
Please make sure that this is ported back to viselements.json...

Reply:
it'll be in my next PR